src/test/ui/lint/lint-ctypes.stderr

```diff
@@ -58,8 +58,17 @@ LL |     pub fn box_type(p: Box<u32>);
    = help: consider adding a `#[repr(C)]` or `#[repr(transparent)]` attribute to this struct
    = note: this struct has unspecified layout
 
+error: `extern` block uses type `std::option::Option<std::boxed::Box<u32>>`, which is not FFI-safe
+  --> $DIR/lint-ctypes.rs:51:28
+   |
+LL |     pub fn opt_box_type(p: Option<Box<u32>>);
+   |                            ^^^^^^^^^^^^^^^^ not FFI-safe
```

Comment:
I'm confused, Option<Box<T>> where T: Sized should also be FFI safe.

Reply:
This test is for `improper_ctypes` - I've not changed the behaviour for that lint at all. `improper_ctypes_definitions`'s behaviour has changed.

Reply:
Oh right, that's probably fine then.